### PR TITLE
Fix error result messages

### LINKED ISSUE #1200: Fix result messages in editor
Filter Radon Error and a format the string that is going to be shown

### PR DESCRIPTION
**Pull request by gabaldon**
It Fixes the error result message that is shown in partial results to be more readable. 

close #1200

### CODE REVIEW

File: src/components/RadonOperator.vue [hunk 1]
```diff
@@ -128,8 +129,9 @@ export default {
       variables: state => state.rad.currentTemplate.variables,
     }),
     radonError() {
+      console.log(this.error)
```
Comment (aesedepece) [inline src/components/RadonOperator.vue:132]: Is this `console.log` intentional?

In any case, we have `console.error` now :fireworks: 
```suggestion
      console.error(this.error)
```

Comment (gabaldon) [reply src/components/RadonOperator.vue:132]: It should be removed anyway, but thanks!

File: src/utils.js [hunk 2]
```diff
@@ -145,6 +145,19 @@ export function isRepeated(key, list) {
...
+// Take the substring between two strings
+export function selectInnerError(string) {
+  if (string) {
+    const substringInBetweenRegex = /(?<=Inner: `Some\()(.*)(?=\))/
+    const matchedArray = string.match(substringInBetweenRegex)
+    if (matchedArray) {
+      return matchedArray[0]
+    } else {
+      return string
+    }
```
Comment (Tommytrg) [inline src/utils.js:157]: Personal opinion

```suggestion
    return matchedArray ? matchedArray[0] : string
```

File: src/utils.js [hunk 3]
```diff
@@ -145,6 +145,19 @@ export function isRepeated(key, list) {
   return list && list[key]
 }
 
+// Take the substring between two strings
+export function selectInnerError(string) {
+  if (string) {
+    const substringInBetweenRegex = /(?<=Inner: `Some\()(.*)(?=\))/
```
Comment (aesedepece) [inline src/utils.js:151]: #RegexPride

**Pull request merged**